plugins/PostProcessingPlugin/scripts/ColorMix.py

```diff
@@ -117,38 +126,44 @@ def execute(self, data):
         secondHeight = 0.0
         firstMix = 0.0
         SecondMix = 0.0
+        modelNumber = 0
```

Comment:
You do not need to declare these initial values. These values will anyway be overriden by the getSettingValueByKey().

Reply:
Implemented